sdk/dataprotection/Azure.ResourceManager.DataProtectionBackup/api/Azure.ResourceManager.DataProtectionBackup.netstandard2.0.cs

```diff
@@ -1509,6 +1580,24 @@ public partial class RangeBasedItemLevelRestoreCriteria : Azure.ResourceManager.
         public string MaxMatchingValue { get { throw null; } set { } }
         public string MinMatchingValue { get { throw null; } set { } }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct RecoveryPointCompletionState : System.IEquatable<Azure.ResourceManager.DataProtectionBackup.Models.RecoveryPointCompletionState>
```

Comment:
this name is quite generic and a bit likely to collide with types in recovery service.
Could you please add some prefixes or suffixes to make this more specific to data protection?